internal/overlord/servstate/handlers.go

```diff
@@ -499,14 +499,18 @@ func getAction(config *plan.Service, success bool) (action plan.ServiceAction, o
 func (s *serviceData) sendSignal(signal string) error {
 	switch s.state {
 	case stateStarting, stateRunning:
+		sig := unix.SignalNum(signal)
```

Comment:
The send-signal functionality already existed via the restart/backoff work; I'm just tweaking the error behavior here.

Reply:
Just to double check, what's the API we intend to offer here?   "HUP", or "SIGHUP"?  I suggest enforcing one of them only, and perhaps only "SIGHUP" here as it's the more formal name, which we're opting not to use in the CLI as we have "signal HUP" already.